internal/ignition/ignition.go

```diff
@@ -1574,6 +1578,12 @@ func (g *installerGenerator) downloadManifest(ctx context.Context, manifest stri
 	if err != nil {
 		return err
 	}
+
+	if len(content) == 0 {
+		// Ignore any empty files.
+		return nil
```

Comment:
I'd prefer for this not to ever actually be present.
Why can't we validate this on the way in and tell the user they're doing something wrong?

Reply:
Sorry, you mean on upload?